README.md

```diff
@@ -1,24 +1,101 @@
 
-## Notes
+1. User fills out the form and hits "Submit", transmitting the form data via
+   a POST request.
+2. **The controller sees that validations have failed, and re-renders the
+   form.**
+3. The view displays the errors to the user.
 
-Really simple readme, given a form and a model with validations, what does a controller action that takes this validation into account look like?
 
-We need to see if the object is valid on form submission, if so, save it, and redirect to the newly created resource.
+## A Note About Page Loads
 
-if not, then we want to re-render the form with the invalid instance in scope so that we can use it to introspect on erorrs (more on that very soon, but we need to know what was wrong.) its important to try to remind the student that the @post from Post#new is long gone - every request happens indepedntly, we've recreated the essence of @post from Post#new when the form was submitted with Post.new(params[:post]). We can't redirect to new to reshow the form because then the invalid @post based on params[:post] is gone. Try to explain this idea. Why we render on invalid objects instead of redirecting.
+You're probably used to validations happening almost instantaneously on websites
+that you interact with on a daily basis. Every browser has a way of visually
+indicating that a page load is happening; in Chrome, for example, the circular
+refresh icon becomes an "X" (for "Stop Loading"), and reverts to refresh when
+the load is complete.
 
-if valid? vs if save on new  - explain the pattern, save returns false and triggers validation
-if update - same with update
+A "page load" means that everything the browser was doing is thrown out, except
+for cookies that are stored between requests. When you get validation feedback
+*without* seeing a page load, that's JavaScript at work, sneakily performing
+requests in the background without throwing away the current page.
 
-very common pattern.
+We won't be using that advanced technique just yet. For now, we'll be performing
+a full page load every time a form is submitted, which means there will be no
+validation feedback for the user until they actually hit the "Submit" button.
 
-<a href='https://learn.co/lessons/validations-in-controller-actions-rails' data-visibility='hidden'>View this lesson on Learn.co</a>
+# Objectives
+
+After this lesson, you'll be able to...
+
+- Define validations on a model
+- Use the validation state of a model in a response conditional in an action
+- Re-render a form with validation errors
+- Validate a create action
+- Validate an update action
+
+# Manually Checking Validation
+
+Up until this point, our `create` action has looked something like this:
+
+```ruby
+# app/controllers/posts_controller.rb
+
+  def create
+    @post = Post.create(post_params)
+
+    redirect_to post_path(@post)
+  end
+```
+
+However, we have two problems now:
+
+1. If the post is invalid, there will be no `show` path to redirect to.
```

Comment:
why will there be no `show` path to redirect to? I'd remind readers here that the post is not persisted to the database, so the generated `post_path` will be incorrect.